assets/script/setup_local_cluster.md

```diff
@@ -13,6 +13,10 @@ btcd --txindex --simnet --rpcuser=kek --rpcpass=kek --datadir=data/btcd/data --l
 
 lnd --rpclisten=localhost:10009 --listen=localhost:10019 --restlisten=localhost:8009 --lnddir=data/lnd --debuglevel=info --bitcoin.simnet --bitcoin.active --bitcoin.node=btcd --btcd.rpcuser=kek --btcd.rpcpass=kek --no-macaroons
 
+# if this is your first time, create the wallet
+lncli --rpcserver=localhost:10009 --no-macaroons --lnddir=data/lnd create
+
+# otherwise unlock the wallet
```

Comment:
This one shouldn't be necessary as the wallet is already create when running `npm run electron-dev`.